### ISSUE TITLE
Next step after bootloader?

### ISSUE THREAD

**Issue by phantomBlurrr** [cmdwtf/Odiin#106]
Hi, I was able to build the bootloader and flash it by first running the task "build bootloader odiin" then connecting the Pitaya Link to the Pitaya Go and running the task "flash bootloader odiin".

Would you mind describing how to continue from here? My understanding is to run the task "build (release UF2) odiin"? But I am unsure. When I run that task or the task "build and flash (release)" I am getting this error:

```
Compiling file: main.cpp
arm-none-eabi-c++: error: unrecognized command line option '-c8'; did you mean '-c'?
make: *** [vendor/nRF5_SDK/components/toolchain/gcc/Makefile.common:276: _build/nrf52840_xxaa/main.cpp.o] Error 1
```

**Comment by phantomBlurrr** [comment]
I edited line 33 in the make file from

`GIT_COMMIT_HASH != git rev-parse HEAD -c8`

to

`GIT_COMMIT_HASH != git rev-parse --short HEAD`

And running the task "build and flash (release) odiin" was able to run and complete all the way through. (Pitaya Link was connected to Pitaya Go during this). Nothing happens when I power on just the Odiin device though. Looking into it...

**Comment by phantomBlurrr** [comment]
Solution is to run the task "build and flash (release)" with the Pitaya Link and Pitaya Go connected up.

**Comment by nitz** [comment]
I'm glad you figured this out before I could even take a look at it!!

~~I took a look at the `git rev-parse` documentation this morning and it seems that the `-c` flag isn't even a thing, so I'm not sure when that was removed. I'll update the `Makefile` so as hopefully no one else runs into this in the future.~~

Great catch!

Edit: Oh it was actually using `head`! Switching to use the built in `--short` in `rev-parse` is a _much_ better solution, as it removes one more dependency on a gnu binary being present on a Windows machine. Thanks a heap for tracking this down and finding a great solution.
